RecoLocalMuon/RPCRecHit/plugins/CSCSegtoRPC.cc

```diff
@@ -87,9 +99,6 @@ std::unique_ptr<RPCRecHitCollection> CSCSegtoRPC::thePoints(const CSCSegmentColl
                     << acos(dz) * 180 / 3.1415926 << " < 135? " << std::endl;
 
         if ((segment->dimension() == 4) && (segment->nRecHits() <= 10 && segment->nRecHits() >= 4)) {
-          //&& acos(dz)*180/3.1415926 > 45. && acos(dz)*180/3.1415926 < 135.){
-          //&& segment->chi2()< ??)Add 3 segmentes in the endcaps???
-
           if (debug)
             std::cout << "CSC \t \t yes" << std::endl;
```

Comment:
LogDebug here and below

Reply:
https://github.com/cms-sw/cmssw/issues/39131

Reply:
I replaced "cout" by "LogDebug"